src/main/java/com/codessquad/qna/Controller/QuestionController.java

```diff
@@ -0,0 +1,49 @@
+package com.codessquad.qna.Controller;
+
+import com.codessquad.qna.domain.Question;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Controller
+public class QuestionController {
+
+    Logger logger = LoggerFactory.getLogger(UserController.class);
+    final private List<Question> questionList = new ArrayList<>();
+
+    @GetMapping("/qna/form.html")
+    public String questionList() {
+        logger.info("askQuestion");
+        return "qna/form";
+    }
+
+    @PostMapping("/qna/questions")
+    public String askQuestion(Question question) {
+        //qList.add();
+        question.setIndex(questionList.size()+1);
```

Comment:
연산자 앞뒤로 한칸씩 공백 잊지 말아주세요!

```suggestion
        question.setIndex(questionList.size() + 1);

Reply:
넵! 말씀해주셔서 감사합니다. 앞으로도 연산자 앞뒤로 공백 한칸씩 꼭 삽입하겠습니다!